### PR TITLE
zh-translation: zh/news/releases/1.11.x/announcing-1.11.5/index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
/zh/news/releases/1.11.x/announcing-1.11.5/index.md

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
cc @Xunzhuo

**Comment by yanrongshi** [comment]
Thanks! @Xunzhuo

File: content/zh/news/releases/1.11.x/announcing-1.11.5/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - /zh/news/announcing-1.11.5
+---
+
+此版本包含一些漏洞修复，从而提高了系统的稳健性。这个版本说明主要描述了 Istio 1.11.4 和 Istio 1.11.5 之间的不同之处。
+
+{{< relnote >}}
+
+## 改变{#changes}
+
+- **修复** 修复了 Istio 部署遵循 `values.pilot.nodeSelector` 参数的问题。
```
Comment (Xunzhuo) [inline content/zh/news/releases/1.11.x/announcing-1.11.5/index.md:18]: Istio -> Istiod

File: content/zh/news/releases/1.11.x/announcing-1.11.5/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  ([Issue #36110](https://github.com/istio/istio/issues/36110))
+
+- **修复** 修复了当 Istio 控制平面有活动的代理连接时，集群内的运维人员不能削减资源的问题。
```
Comment (Xunzhuo) [inline content/zh/news/releases/1.11.x/announcing-1.11.5/index.md:21]: Operator is a concept in Kubernetes, it does not mean `运维人员`, please just keep it.

File: content/zh/news/releases/1.11.x/announcing-1.11.5/index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  ([Issue #35657](https://github.com/istio/istio/issues/35657))
+
+- **修复** 修复了通过添加补丁版本来获取释放污点 URL 的问题。
+
+- **修复** 修复了在 Envoy 中当通过 `MeshNetworks` 配置多网络网关时， `LbEndpointValidationError.LoadBalancingWeight: value must be greater than or equal to 1` 的问题。
+
+- **修复** 修复了在 k8s 1.21+ 版本时，工作负载名称标准标签没有正确填充 `CronJob` 的问题。
```
Comment (Xunzhuo) [inline content/zh/news/releases/1.11.x/announcing-1.11.5/index.md:28]: `metric labels` is better to translate into `监控指标标签` not `标准标签`

**Review by Xunzhuo — CHANGES_REQUESTED** [review]

File: content/zh/news/releases/1.11.x/announcing-1.11.5/index.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+## 改变{#changes}
+
+- **修复** 修复了 Istio 部署遵循 `values.pilot.nodeSelector` 参数的问题。
+  ([Issue #36110](https://github.com/istio/istio/issues/36110))
+
+- **修复** 修复了当 Istio 控制平面有活动的代理连接时，集群内的运维人员不能削减资源的问题。
+  ([Issue #35657](https://github.com/istio/istio/issues/35657))
+
+- **修复** 修复了通过添加补丁版本来获取释放污点 URL 的问题。
```
Comment (Xunzhuo) [inline content/zh/news/releases/1.11.x/announcing-1.11.5/index.md:24]: tar URL does not mean `污点 URL`, it is a way to release istio like `https://github.com/istio/istio/releases/download/1.7.0/istio-1.7.0-linux-amd64.tar.gz`, this is a tar URL

**Comment by yanrongshi** [comment]
@Xunzhuo，Hey, check it out for me，boss

**Review by Xunzhuo — APPROVED** [review]

**Comment by yanrongshi** [comment]
@Xunzhuo Is this have a  issue? boss

**Pull request merged**